### ISSUE TITLE
Unicode in Source files

### ISSUE THREAD

**Issue by gunterkoenigsmann** [wxMaxima-developers/wxmaxima#1691]
@daute and @richardgobeli  have found out: If there is no BOM at the end of a source file it is up to the C compiler to decide if the file is unicode or an exotic codepage => We need to replace all 8-bit letters by an "\uxxxx" command that explicitly specifies that this is meant as an Unicode char.

**Comment by gunterkoenigsmann** [comment]
A regex that seems to find this kind of characters is

````
"[\x80-\xFF]"
````

**Comment by gunterkoenigsmann** [comment]
Grep doesn't like that one if not in the form

````
grep --color -nH --null -P -n -e "[\x80-\xFF]" * -R
````

**Comment by richardgobeli** [comment]
What does the & character do in the string in front of ad?
series_sub->Append(menu_pade, _("P&ad\u00E9 Approximation..."),

There must be more to it than just putting \u00E9 because the character is still wrong from yesterday.
![image](https://user-images.githubusercontent.com/31253976/173247821-6e1c8a05-072b-4609-81a8-0fd67ead3ef2.png)
![image](https://user-images.githubusercontent.com/31253976/173247839-c15ddfb5-e5bc-4614-a481-0b07af71d375.png)

**Comment by gunterkoenigsmann** [comment]
The "&" character makes that character an accellerator (I believe you can press that key or alt + that key) for reaching the menu entry faster.

But that unicode still isn't treated correctly is weird...

**Comment by daute** [comment]
Fixed with commit 5cbec073ee8cb16750bbe39986a4f3ee97378a20.
